docs/BigQueryTable-batchsink.md

```diff
@@ -53,6 +53,11 @@ is ignored if the table already exists.
 * When this is set to true, table will be created with required partition filter. 
 * When this is set to false, table will be created without required partition filter.
 
+**Clustering Order**: List of fields that determines the sort order of the data. Fields must be of type 
+INT, LONG, STRING, DATE, TIMESTAMP, BOOLEAN or DECIMAL. Maximum number of fields for clustering order 
```

Comment:
change "Maximum number of fields for clustering order should not be more that 4" to:

"Tables cannot be clustered on more than 4 fields."